README.md

```diff
@@ -1,16 +1,21 @@
-<p align="center">
-  <a href="https://glide-core.crowdstrike-ux.workers.dev">
-    <picture alt="Glide Core Logo">
-      <source srcset="https://github.com/CrowdStrike/glide-core/blob/main/.github/logos/light.png?raw=true" media="(prefers-color-scheme: light)" />
-      <source srcset="https://github.com/CrowdStrike/glide-core/blob/main/.github/logos/dark.png?raw=true" media="(prefers-color-scheme: dark)" />
-      <img src="https://github.com/CrowdStrike/glide-core/blob/main/.github/logos/light.png?raw=true" width="500" />
-    </picture>
-  </a>
-</p>
-
-<p align="center"> 
-  A Web Component design system from CrowdStrike.
-</p>
+<div align="center">
+  <p align="center">
+    <a href="https://glide-core.crowdstrike-ux.workers.dev">
+      <picture alt="Glide Core Logo">
+        <source srcset="/.github/logos/light.png?raw=true" media="(prefers-color-scheme: light)" />
```

Comment:
Getting real-time coverage to show up is a whole thing and would require us to use a third-party service. I figured a hardcoded badge will do considering our thresholds.

Reply:
Wonder if this will work on npm's site as well?

Reply:
I believe it will. Svelte [paths](https://github.com/sveltejs/svelte/blob/main/README.md?plain=1#L4) similarly and their logo [works](https://www.npmjs.com/package/svelte) on NPM.